test/src/test/java/SslTest.java

```diff
@@ -0,0 +1,24 @@
+import io.restassured.RestAssured;
+import org.hamcrest.core.IsNull;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.*;
+
+
+public class SslTest extends AbstractFeatureServiceTest {
+
+    @Before
+    public void connectToSslPort() {
+        RestAssured.port = 443;
+        RestAssured.baseURI = "https://localhost";
+    }
+
+    @Test
+    public void getServiceDescriptor() {
```

Comment:
Can you also add this test? It ensures that SSL is required, as using the "relaxed" validation doesn't totally verify that SSL is required and setup correctly.

```
    @Test
    public void withoutRelaxedHTTPSValidation() {
        SSLHandshakeException ex = assertThrows(SSLHandshakeException.class,
            () -> getRequest(request2path("featureService.json")));
        
        assertTrue(
            "The request should fail because the Koop server requires SSL, and it's using a self-signed cert that " +
                "the underlying JVM is not expected to trust; actual message: " + ex.getMessage(),
            ex.getMessage().contains("PKIX path building failed")
        );
    }
```